### DISCUSSION TITLE
Illustration with a custom:mini-graph-card - Lower part gone?

### DISCUSSION THREAD

**Discussion by barthel-eu** [gillesvs/librelink#23] [Q&A]
After installation, I only changed the name in the entity to mine and it shows nicely the upper part, but the curve on the lower part is completely missing?
![image](https://github.com/user-attachments/assets/338a892f-289e-47a9-a467-5da45f459b40)

Side issue, the times should be 24-hour clock as everywhere else in my HA installation? Checked in the user settings, set to 24h 😳

**Comment by gillesvs** [comment]
Hello,
I think this is more an issue with the card usage than with the Librelink integration

**Comment by barthel-eu** [comment]
Different from the App, it seems that the curve only starts to "fill" after the card has been created, the previous data ain't shown initially. By coincidence I just checked on my "test-board" and found it showing data now.
It's still somewhat disturbing that the time is shown using 12-hour-clock. Will see now, if I can trigger some added automation.
